modules/wsl-distro.nix

```diff
@@ -76,8 +76,11 @@ in
 
       extraPackages = mkIf cfg.useWindowsDriver [
         (pkgs.runCommand "wsl-lib" { } ''
-          mkdir "$out"
-          ln -s /usr/lib/wsl/lib "$out/lib"
+          mkdir -p "$out/lib"
+          # we cannot just symlink the lib directory because it breaks merging with other drivers that provide the same directory
+          for so in libcuda.so libd3d12.so libd3d12core.so libdxcore.so; do
```

Comment:
Do we need other cuda so's? What are the amd names?

Reply:
AMD does not support CUDA

Reply:
I meant other shared objects in general, like the amd one. What's the name for it? Is there even one?